wordpress/wp-content/plugins/memberful-wp/src/contrib/ad-providers/raptive-ads.php

```
@@ -0,0 +1,53 @@
+    return is_plugin_active( 'adthrive-ads/adthrive-ads.php' );
+  }
+
+  public function get_name() {
+    return $this->name;
+  }
+
+  public function get_identifier() {
+    return $this->identifier;
+  }
+
+  public function disable_ads_for_user($user_id) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$user_id'. (undefined)

(UnusedFormalParameter)